db.py

```diff
@@ -78,7 +78,23 @@ def shiftRanking(user, nick, operation, amount):
         else:
             c.execute(f"UPDATE rankings SET ranking = ranking - 1 WHERE user={user} AND ranking<={newRank} AND ranking>{oldRank}")
         c.execute(f"UPDATE rankings SET ranking = {newRank} WHERE user={user} AND member='{name}'")
-        
+
+        db.getdb().commit()
+
+    def swapRanks(user, member1, member2):
+        c = db.getdb().cursor()
+        name1 = nickToName(member1)
```

Comment:
these functions don't belong in the db script. do the validation outside. you can refer to how it was done for the stats command